packages/react/src/select/positioner/useSelectPositioner.ts

```diff
@@ -15,9 +15,9 @@ import { useScrollLock } from '../../utils/useScrollLock';
 export function useSelectPositioner(
   params: useSelectPositioner.Parameters,
 ): useSelectPositioner.ReturnValue {
-  const { open, alignItemToTrigger, mounted, triggerElement } = useSelectRootContext();
+  const { open, alignItemToTrigger, mounted, triggerElement, modal } = useSelectRootContext();
 
-  useScrollLock(alignItemToTrigger && mounted, triggerElement);
+  useScrollLock(modal && open, triggerElement);
```

Comment:
Is `alignItemToTrigger=true` always forced to be `modal` mode? It doesn't support non-scroll locked mode, even if they wanted it

Reply:
OK, I'll make it `(alignItemToTrigger || modal) && open`, then